src/utils/storage/index.ts

```diff
@@ -18,15 +18,18 @@ enum StorageKeysEnum {
   OtherProfileData = 'otherprofiledata',
   LastPromptNotification = 'lastPromptNotification',
   LastSelectedMenu = 'lastSelectedMenu',
-  chatInfo = 'chatInfo'
+  chatInfo = 'chatInfo',
+  BlockedStatus = 'blockedStatus',
+  BlockingStatus = 'blockingStatus',
+  Feed = 'feed'
 }
 
 interface IStorage {
-  get: () => string;
+  get: () => string | undefined;
   set: (value: string) => void;
   clear: () => void;
   setForKey: (key: string, value: string) => void;
-  getForKey: (key: string) => string;
+  getForKey: (key: string) => string | undefined;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change in the return type of `get` and `getForKey` methods from `string` to `string | undefined` is a good one. It allows for better handling of cases where the key does not exist in the storage, which was not possible before when the return type was just `string`.



<!-- This is an auto-generated comment by OSS CodeRabbit -->